src/mgr/DaemonState.h

```diff
@@ -45,17 +45,41 @@ class PerfCounterInstance
     {}
   };
 
+  class AvgDataPoint
```

Comment:
Would it make sense to inherit from DataPoint here?

Reply:
I'm okay with avoiding inheritance for simple types like this.